templates/job-init-db-schema.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if gt (.Values.zabbixserver.replicaCount | int) 1 }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "zabbix.fullname" . }}-init-db-schema
+  labels:
+    app: {{ template "zabbix.fullname" . }}-init-db-schema
+    app.kubernetes.io/name: init-db-schema
+    helm.sh/chart: {{ include "zabbix.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}-init-db-schema
+    app.kubernetes.io/managed-by: {{ .Release.Service }}-init-db-schema
+spec:
+  template:
+    spec:
+      containers:
+      - name: init-db-schema
+        {{- if .Values.zabbix_image_tag }}
+        image: {{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbix_image_tag }}
+        {{- else }}
+        image: {{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbixserver.image.tag }}
```

Comment:
With tag centralized, this line and the if can be removed, correct?

Reply:
commit 3f529c7